spec/models/poll/web_vote_spec.rb

```diff
@@ -119,5 +156,35 @@
         expect(Poll::Answer.count).to be 2
       end
     end
+
+    context "Open-ended questions" do
```

Comment:
What do you think about adding one more test, for the case when the open answer already exists and we're sending a new open answer? The test would check that there's only one answer in the database and it's got the new text.